spec/features/admin_merchants/index_spec.rb

```diff
@@ -45,6 +45,7 @@
         expect(page).to have_button("Disable")
 
         click_button("Disable")
+        expect(yeasty.reload.status).to eq("Disabled")
```

Comment:
nice use of reload! During our refactor, we should go back through and see about adding this to Taryn's `merchant enable/disable`